src/context/extensions/steps/getExtensions.js

```diff
@@ -191,6 +192,19 @@ function getCompleteExtensionTree(type, roc, path, initialState) {
     );
 }
 
+function getNearestParents(roc) {
+    return [
+        ...(roc.packages || []),
+        ...(roc.plugins || []),
+    ].map((parent) => {
```

Comment:
Right, in a sense yes since we use the paths that we have in `packages` and `plugins` with Node.js default resolving algorithm. We have not implemented our own version of this here however.